statistics/integration_test.go

```diff
@@ -499,7 +498,6 @@ func TestAnalyzeLongString(t *testing.T) {
 }
 
 func TestOutdatedStatsCheck(t *testing.T) {
-	domain.RunAutoAnalyze = false
```

Comment:
Since the default value of `RunAutoAnalyze` is `true`, I think simply removing it may cause some trouble. @chrysan PTAL.

Reply:
It's false in the test suite. See the change in session/bootstrap.go:2012.

Reply:
Prefer to keep the variable set in ut or at least an assertion to make the ut safe no matter what has been changed in variable module.

Reply:
Agreed, an explicit update for UT is more solid. @Alkaagr81 could you update this?

Reply:
The problem with setting it in the test is:
- If you update the atomic directly, it can get overwritten when the sysvar cache is updated. This is because the sysvar cache calls the SetGlobal func with the value from the mysql.global_variables.
- To safely change it, tk.MustExec("set global sysvar=x") must be used. But tk is not available in all of the tests. We could add it, but it's quite a bit more code to set up each time.

This is why it uses the bootstrap to disable auto-analyze instead.

Reply:
We can however add an assertion that runautoanalyze is false. Reading the atomic value is no problem.

Reply:
I see, an alternative way is to create it as another 'enhancement' issue as a backlog, then we could make this PR delivered with the next version(v6.1)

Reply:
Forked to https://github.com/pingcap/tidb/issues/34792